include/openthread/border_agent.h

```diff
@@ -110,6 +110,30 @@ typedef enum otBorderAgentState
     OT_BORDER_AGENT_STATE_ACTIVE  = 2, ///< Border agent is connected with external commissioner.
 } otBorderAgentState;
 
+typedef struct otBorderAgentCounters
+{
+    uint64_t mEpskcActivations;
+    uint64_t mEpskcDeactivationClears;
```

Comment:
Add documentation? A short description for each counter to indicate what it tracks?

Reply:
Added documentation.